src/industry_cmd.cpp

```diff
@@ -2061,8 +2061,10 @@ CommandCost CmdBuildIndustry(DoCommandFlag flags, TileIndex tile, IndustryType i
 					}
 					if (ret.Succeeded()) break;
 				}
+				cur_company.Restore();
+			} else {
+				ShowErrorMessage(STR_ERROR_CAN_T_GENERATE_INDUSTRIES, STR_ERROR_PROSPECTING_INDUSTRY_FAILED, WL_INFO);
```

Comment:
The error message could use some work, especially the first line ("Can't generate industries").

I suggest the following error message:
```
Can't prospect industry...
Prospecting was unlucky and failed; try again
```

Reply:
Both suggestions implemented. (Is it right to hit "Resolve Conversation" on GitHub when I think the changes suggested have been added? Or is this usually done by the reviewers?)

Reply:
Yes, you can Resolve Conversation when you've implemented the suggestion. 🙂